pkgs/tools/graphics/exifprobe/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "exifprobe";
+  version = "unstable-2018-06-19";
+
+  src = fetchFromGitHub {
+    owner = "hfiguiere";
+    repo = "exifprobe";
+    rev = "ce1ea2bc3dbbe8092b26f41cd89831cafe633d69";
```

Comment:
Can you comment why you choose this rev? Is it a tag or just the latest of today?

Reply:
The 2.0.1 release was in November 2013, so it seems most appropriate to use the latest commit.